constraints.txt

```diff
@@ -64,7 +64,7 @@ semantic-version==2.6.0
 semver==2.8.1
 simplegeneric==0.8.1
 six==1.11.0
-structlog==18.2.0
+structlog==20.1.0
```

Comment:
see [this thread](https://github.com/rsksmart/lumino/pull/120#discussion_r489418906)